ios.toolchain.cmake

```
@@ -334,6 +335,30 @@ elseif(PLATFORM_INT STREQUAL "OS64COMBINED")
   else()
     message(FATAL_ERROR "Please make sure that you are running CMake 3.14+ to make the OS64COMBINED setting work")
   endif()
+elseif(PLATFORM_INT STREQUAL "OS64COMBINED_ASI")
+  set(SDK_NAME iphoneos)
+  if(MODERN_CMAKE)
+    if(NOT ARCHS)
+      if (XCODE_VERSION_INT VERSION_GREATER 10.0)
```

```suggestion
      if (XCODE_VERSION_INT VERSION_GREATER_EQUAL 12.0)
```

ability to build for arm64 simulator appeared only in Xcode 12